src/docker/custom.rs

```diff
@@ -128,6 +128,10 @@ impl<'a> Dockerfile<'a> {
             }
         }
 
+        // note that this is always relative to the PWD: if we have `/Dockerfile`
+        // (relative to the workspace root), then running a build in `/src`
+        // requires the Dockerfile path to be specified as `../Dockerfile`,
+        // and running from `/..` requires `${PACKAGE_NAME}/Dockerfile`.
         docker_build.args(["--file".into(), path]);
```

Comment:
having issues parsing this, maybe

```suggestion
        // note that this is always relative to the PWD: if we have `$root/Dockerfile`
        // (relative to the workspace root), then running a build
        // `PWD=$root/src/ cross build`
        // would require the Dockerfile path to be specified as `../Dockerfile`.
        docker_build.args(["--file".into(), path]);
```

Reply:
If we use this with the `$root`, I think we can change it to `$workspace_root` and then remove `(relative to the workspace root)`.